src/store/conversationsStore.js

```diff
@@ -19,6 +19,7 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+import { isEmpty } from 'lodash'
```

Comment:
**issue (blocking):** Lodash does not support tree shaking.

```js
import { isEmpty } from 'lodash'
``` 
adds about **71.7kB** to bundle.

```js
import isEmpty from 'lodash/isEmpty.js'
```
would add **6.7kB** and actually adds **0.14kB** because we already have it with `isEqual`.

```suggestion
import isEmpty from 'lodash/isEmpty.js'
```